### ISSUE TITLE
`Named export not found error` with upgrade to Vite 5 because of lexical packages still being CJS (instead of ESM)

### ISSUE THREAD

**Issue by umaranis** [umaranis/svelte-lexical#68]
**Sample error - this error occurs on every import from a lexical package**

Error when evaluating SSR module /@fs/Users/umar/Documents/Projects/lexical/svelte-lexical/packages/svelte-lexical/src/components/toolbar/BlockFormatDropDown/HeadingDropDownItem.svelte: failed to import "@lexical/selection"
|- SyntaxError: [vite] Named export '$setBlocksType' not found. The requested module '@lexical/selection' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@lexical/selection';
const {$setBlocksType} = pkg;

    at analyzeImportedModDifference (file:///Users/umar/Documents/Projects/lexical/svelte-lexical/node_modules/.pnpm/vite@5.0.11/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:50953:19)
    at nodeImport (file:///Users/umar/Documents/Projects/lexical/svelte-lexical/node_modules/.pnpm/vite@5.0.11/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:50904:9)
    at async ssrImport (file:///Users/umar/Documents/Projects/lexical/svelte-lexical/node_modules/.pnpm/vite@5.0.11/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:50799:24)
    at async eval (/Users/umar/Documents/Projects/lexical/svelte-lexical/packages/svelte-lexical/src/components/toolbar/BlockFormatDropDown/HeadingDropDownItem.svelte:8:31)
    at async instantiateModule (file:///Users/umar/Documents/Projects/lexical/svelte-lexical/node_modules/.pnpm/vite@5.0.11/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:50861:9)

**Comment by umaranis** [comment]
Vite believes it is not their issue. NodeJS is not able to resolve the CJS module, so Vite5 can't do anything about it.

https://github.com/vitejs/vite/issues/14332

**Comment by umaranis** [comment]
There is an open `lexical` issue to move it from CJS to ESM
https://github.com/facebook/lexical/issues/1707

**Comment by umaranis** [comment]
A workaround to resolve this issue is to change the import statement from
```
import {exportFile} from '@lexical/file';
```
to 
```
import pkgfile from '@lexical/file';
const {exportFile} = pkgfile;
```

**Comment by umaranis** [comment]
resolved with 0.4.0
